node-require/add.js

```diff
@@ -0,0 +1,6 @@
+const add = (num1, num2) => {
+  let sum = num1 + num2;
+  console.log('result: ', sum)
```

Comment:
Try updating your functions to `return` their result instead of console logging it. You can do the logging in `calculate.js`.